### PR TITLE
[Contract] add searchable dropdown menu for mobile view

### PR DESCRIPTION
**Pull request by SamuelQZQ**
Could be tested with this link:
https://deploy-preview-510--aptos-explorer.netlify.app/account/0x1/modules/code/block/get_current_block_height?network=devnet&feature=dev

In View tab:
image

In Code tab:
image

### CODE REVIEW

**Review by 0xbe1 — APPROVED** [review]
lgtm

**Comment by SamuelQZQ** [comment]
@jillxuu  FYI, to make it easier to review and avoid merge conflict with #509, currently we make this PR based on the #509 branch. So before merging this PR, we should change the base back to the main branch. Or you can leave it to me to merge it.

**Review by jillxuu — APPROVED** [review]
one small nit not relevant to this PR: can we actually remove "COPY CODE" and expand button when code's not available? doesn't look the best on the dark mode.

**Pull request merged**
